### ISSUE TITLE
fix: fast-menu component seems to be broken in latest release on unpkg (13/7/21)

### ISSUE THREAD

**Issue by Valks** [microsoft/fast#4957]
# 🐛 Bug Report

When using the fast-menu component the layout is completely broken.

## 💻 Repro or Code Sample
```html
<html>
    <body>
        <fast-design-system-provider use-defaults>
            <fast-menu>
                <fast-menu-item>
                    Menu item 1
                    <fast-menu slot="submenu">
                        <fast-menu-item>Menu item 1.1</fast-menu-item>
                        <fast-menu-item>Menu item 1.2</fast-menu-item>
                        <fast-menu-item>Menu item 1.3</fast-menu-item>
                    </fast-menu>
                </fast-menu-item>
                <fast-menu-item>
                    Menu item 2
                    <fast-menu slot="submenu">
                        <fast-menu-item>Menu item 2.1</fast-menu-item>
                        <fast-menu-item>Menu item 2.2</fast-menu-item>
                        <fast-menu-item>Menu item 2.3</fast-menu-item>
                    </fast-menu>
                </fast-menu-item>
                <fast-menu-item>
                    Menu item 3
                    <fast-menu slot="submenu">
                        <fast-menu-item>Menu item 3.1</fast-menu-item>
                        <fast-menu-item>Menu item 3.2</fast-menu-item>
                        <fast-menu-item>Menu item 3.3</fast-menu-item>
                    </fast-menu>
                </fast-menu-item>
            </fast-menu>
        </fast-design-system-provider>
        <script type="module" src="https://unpkg.com/@microsoft/fast-components"></script>
    </body>
</html>
```
## 🤔 Expected Behavior

Expected a menu that expands out correctly as you highlight each item.

## 😯 Current Behavior

It's broken in a few ways, a picture probably helps.
![image](https://user-images.githubusercontent.com/3953780/125377220-f13d6480-e3cf-11eb-97f8-ddc4c866d0cc.png)

## 🔦 Context

As you can imagine I would like to use a menu. So yeah.
As a small aside. What is the recommendation when the menu item should be a link? If you place an anchor in an item it doesn't fill the container as might be (hoped) expected.

## 🌍 Your Environment

* OS & Device: Windows on PC
* Browser Microsoft Edge
* Version 91.0.864.67

**Comment by chrisdholt** [comment]
@scomea I believe this was fixed with #4980, is that corrrect?

**Comment by scomea** [comment]
Yes, that should fix the layout within a menu item with a submenu and no icons other than the expand toggle which is clearly going on here.  I'm wondering if there is something else going on with the submenu being under the parent menu's content though.

**Comment by chrisdholt** [comment]
Closed by #4980
